package.json

```diff
@@ -61,7 +61,9 @@
     "@ethereumjs/common": "3.0.1",
     "@ethereumjs/tx": "4.0.1",
     "@ethereumjs/util": "8.0.2",
+    "@ethersproject/abi": "5.7.0",
```

Comment:
what's up w. all the additional dependency requirements in the linting PRs?

Reply:
These are all extraneous deps, i.e. we are using them without them being in the package json.

Reply:
cool, makes sense